crates/sequencer-client/src/lib.rs

```diff
@@ -99,7 +100,11 @@ impl From<GetSoftBatchResponse> for SignedSoftConfirmationBatch {
             val.da_slot_txs_commitment,
             val.pre_state_root,
             val.l1_fee_rate,
-            val.txs.unwrap_or_default(),
+            val.txs
```

Comment:
is there any way we can get rid of this?

if any other client queries that endpoint it will look like this:

```json
{
  ...
  txs: [
    { tx: "" },
    { tx: "" }
  ]
}
```

Maybe if we define HexTx like this it won't get serialized that way:
```rust
pub struct HexTx(Vec<u8>);
```

Reply:
anyone that sends an http request :)

Reply:
checked it now works as expected